platform/core/crates/common/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "common"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Bump edition up?

Reply:
AFAIK, 2022 edition is not yet available.